client/src/components/brushableHistogram/index.js

```diff
@@ -385,7 +385,13 @@ class HistogramBrush extends React.Component {
       <div
         id={`histogram_${field}`}
```

Comment:
I am not sure `field` is guaranteed to be anything other than a string, which means it could contain characters that are not legal in an element ID.    Might consider escaping these for the (rare) case we get a dataset with fields such as "foo bar"

Reply:
actually, now that I look more, I think this is definitely needed - at a minimum space needs to be handled, as it is not legal in an ID, and seems to be legal (and not rare) as a column name.

Reply:
Will fix (but for the record, I didn't add this :)

Reply:
lol, sorry!   And I'm only really concerned about places that are really going to cause a problem with legit user data.  So I think only `id` requires the space-escape.